### PR TITLE
[CBRD-24087]  Improvement of oid_compare() function and comparison logic of codes using this function.

### PR DESCRIPTION
**Pull request by ctshim**
http://jira.cubrid.org/browse/CBRD-24087

Purpose
The oid_compare() function is called when comparing OIDs in various places in the source code.
In particular, it is designated as a comparison function such as sort (qsort) and search (lsearch).
Since very frequent calls occur, we want to increase the efficiency of the code.

Implementation
  Remove unnecessary substitution statements

Remarks
N/A

### CODE REVIEW

File: src/storage/oid.h [hunk 1]
```diff
@@ -92,6 +92,7 @@
     (oidp)->volid = NULL_VOLID; \
   } while(0)
 
+#define oid_is_equal_value(oid1, oid2)  OID_EQ((oid1), (oid2))
```
Comment (hgryoo) [inline src/storage/oid.h]: Do we need this macro?

Comment (ctshim) [reply src/storage/oid.h]: I can't say it's absolutely necessary.
However, I tried to make the macro name more explicit while defining a macro that replaces the existing code of the form oid_cpmpare() == 0.

Comment (hgryoo) [reply src/storage/oid.h]: If it only calls OID_EQ, this macro is redundant. Also, the name of C macro function should be capital by naming convention.

Comment (ctshim) [reply src/storage/oid.h]: @hgryoo OK, I will fix it.

**Review by beyondykk9 — APPROVED** [review]

**Review by hyunikn — APPROVED** [review]

**Review by youngjinj — APPROVED** [review]

**Review by shparkcubrid — APPROVED** [review]

**Review by hgryoo — APPROVED** [review]

**Pull request merged**